### PR TITLE
Testing - first PR

### PR DESCRIPTION
**Pull request by DeMiro5001**
## Problem

- *First PR, used yologen*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by DeMiro5001** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -2,135 +2,158 @@
...
     main.url = "/"
 
-    [resources.ports]
-    # This will pick a random port for reverse-proxying and store it as the $port setting
-
     [resources.apt]
-    # This will automatically install/uninstall the following apt packages
-    # and implicitly define the $phpversion setting as 8.0 (if phpX.Y-foobar dependencies are listed)
-    packages = "mariadb-server, deb1, deb2, php8.0-foo, php8.0-bar"
+    packages = "mariadb-server, php8.3-mysqli, php8.3-mbstring, php8.3-xml, php8.3-intl"
```
Comment (tituspijean) [inline manifest.toml:156]:
```suggestion
    packages = "mariadb-server, php8.3, php8.3-mysqli, php8.3-mbstring, php8.3-xml, php8.3-intl"
```

[The CI](https://ci-apps-bookworm-dev.yunohost.org/ci/job/3275) complains about `phpversion` not being set. Let's try to properly declare it from the APT packages.

**Comment by DeMiro5001** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**